### PR TITLE
[Mod Center] Add feedback link to mod's Community Mod Chat Channel

### PR DESCRIPTION
**Pull request by msarit**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Adds a feedback link to mod's Community Mod Chat Channel

Screen Shot 2020-06-17 at 11 02 53 AM

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help
- [X] no, not yet

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [X] no documentation needed

### CODE REVIEW

File: app/views/moderations/_mod_sidebar_left.html.erb [hunk 1]
```diff
@@ -40,4 +34,17 @@
...
+  <% if @community_mod_channel %>
+    <div class="fs-s mod-feedback">
+      <span class="fw-medium">Have feedback to improve your Mod experience?</span>
+      <span>Please share in
+        <a
+          class="mod-feedback-link"
+          href="/connect/<%= @community_mod_channel.slug %>"
+          target="_blank"
+          rel="noopener noreferrer nofollow">
+          <%= @community_mod_channel.channel_name %></a>.
```
Comment (Zhao-Andy) [inline app/views/moderations/_mod_sidebar_left.html.erb:46]: I think we can do "your Connect Channel!" in plain text here with no link, and then in the "Resources" section we have a link to "Connect Channel" which would link to this specific channel.

Comment (msarit) [reply app/views/moderations/_mod_sidebar_left.html.erb:46]: ok!

File: app/views/moderations/_mod_sidebar_left.html.erb [hunk 2]
```diff
@@ -21,8 +21,8 @@
   </div>
   <div class="fw-bold fs-s mod-center-section-title resources">RESOURCES</div>
   <div id="mod-center-other-options">
-    <a href="#" class="crayons-link crayons-link--block">
-      Feedback?
+    <a href="/connect/<%= @community_mod_channel.slug %>" target="_blank" rel="noopener noreferrer nofollow" class="crayons-link crayons-link--block">
```
Comment (Zhao-Andy) [inline app/views/moderations/_mod_sidebar_left.html.erb:24]: Need an `if` statement around this anchor link

Comment (msarit) [reply app/views/moderations/_mod_sidebar_left.html.erb:24]: 🤦

**Pull request merged**